docs/src/administration/cli/_index.md

```diff
@@ -174,12 +174,33 @@ platform mount:download --all --app app --target local-backup
 ### Autocomplete commands
 
 The CLI provides tab autocompletion for commands, options, and some values (your projects, valid regions).
+To enable autocompletion, follow this step:
 
-Your system must include the `bash-completion` package or an equivalent.
-This isn't available by default on macOS, but can be installed via `brew`.
-Check your home directory and ensure that the file `~/.platformsh/autocompletion.sh` is being included by your shell.
+{{< codetabs >}}
++++
+title=Using Homebrew
+highlight=false
+file=none
++++
 
-If you experience issues, run `platform self:install` to attempt a reinstall of this utility.
+Follow the [Homebrew documentation on shell completion](https://docs.brew.sh/Shell-Completion).
+
+<--->
++++
+title=Manually
+highlight=false
+file=none
++++
+
+Add the following to your shell's startup (`.bashrc`, `.zshrc`, or the equivalent):
+
+```bash
+eval $(platform completion)
+```
+
+This requires the `bash-completion` package to be installed.
```

Comment:
When I ran `platform completion`, the last line was this:
```
>&2 echo "The 'bash-completion' package is required but doesn't appear to be installed.";
```
But I haven't tested it out or anything.

Reply:
This is a warning that will be displayed if the needed package is not installed

```
if [ "$(type -t _get_comp_words_by_ref)" == "function" ]; then
    complete -F _platform_fc75f55f6a855676_complete "platform";
else
    >&2 echo "Completion was not registered for platform:";
    >&2 echo "The 'bash-completion' package is required but doesn't appear to be installed.";
fi;
```

Since this is already handled by the script, I'd propose we leave it out of the docs to avoid confusion.